src/app/file-manager/reducers/transferables.reducer.ts

```diff
@@ -38,6 +38,58 @@ export class TransferableState {
         return this.items.filter(x => x.type === Type.UPLOAD).length;
     }
 }
+function sortList(items) {
+    items.sort((a, b) => {
+        switch (a.status) {
+          case ItemStatus.DOWNLOADING:
+          {
```

Comment:
pretty sure typescript doesn't require a `{` `}` around the case block here, e.g.
```
case ItemStatus.DOWNLOADING:
    if (b.status === a.status) {
        ...
```